src/platforms/numpy.rkt

```diff
@@ -53,52 +53,52 @@
 
 (define cost-model
```

Comment:
There's one cost that's missing and I think that's a "move" cost, as in, the cost of `(FPCore () 1)`. I believe the tuning script prints this as "baseline" or something. These should be used for "terminal" costs.

Reply:
Ok, I added it.

Reply:
The boolean default cost should probably be a move. Or something like a move

Reply:
Ok, should the boolean if cost still be 1?

Reply:
Or are we making it use sum?

Reply:
Boolean cost should probably be the move cost. At least less than or equal